source/includes/_invoice.md

```diff
@@ -22,56 +22,62 @@ An invoice can have one of the following status values: DRAFT, COMMITTED, or VOI
-| **quantity** | number | user or system | Rate associated with the `Plan` |
-| **currency** | string | user or system | Currency associated with the account |
-| **itemDetails** | string | user or system | In usage mode, details about what was invoiced. |
-| **childItems** | list | user or system | In the hierarchical model, the items for the children. |
+| Name                     | Type     | Generated by   | Description                                                                            |
+|--------------------------|----------|----------------|----------------------------------------------------------------------------------------|
+| **invoiceItemId**        | string   | system         | UUID for the invoice item |
+| **invoiceId**            | string   | system         | UUID for the invoice |
+| **linkedInvoiceItemId**  | string   | system         | UUID for the linked item, if any (see below)|
+| **accountId**            | string   | system         | UUID for the account|
+| **childAccountId**       | string   | system         | In the hierarchical model, the UUID of the child account|
+| **bundleId**             | string   | system         | UUID for the bundle|
+| **subscriptionId**       | string   | system         | UUID for the subscription (present only if invoice item corresponds to a subscription) |
+| **productName**          | string   | system         | Name of the `Product` for this subscription if any |
+| **planName**             | string   | system         | Name of the `Plan` for this subscription if any  |
+| **phaseName**            | string   | system         | Name of the `PlanPhase` for this subscription if any  |
+| **usageName**            | string   | system         | Name of the `Usage` section for this subscription if any |
+| **prettyProductName**    | string   | system         | Pretty name of the `Product` for this subscription if any |
+| **prettyPlanName**       | string   | system         | Pretty name of the `Plan` for this subscription if any|
+| **prettyPhaseName**      | string   | system         | Pretty name of the `PlanPhase` for this subscription if any |
+| **prettyUsageName**      | string   | system         | Pretty name of the `Usage` section for this subscription if any|
+| **itemType**             | string   | system         | Item type (see below)  |
+| **description**          | string   | user or system | Optional description of the item|
+| **startDate**            | date     | user or system | Start date of the period invoiced  |
+| **endDate**              | date     | user or system | End date of the period invoiced    |
+| **amount**               | number   | user or system | Amount being invoiced |
+| **rate**                 | number   | user or system | Rate associated with the `Plan`|
+| **currency**             | string   | user or system | Currency associated with the account|
+| **quantity**             | number   | TODO           | TODO|
```

Comment:
Just FYI - still some TODO left.

Reply:
Yes, I wasn't sure what the `quantity` field refers to, I could not find it easily in the code either. Do you remember? Otherwise, I will dig deeper into the code.

Reply:
To be double checked, but I believe it can used in different contexts:

* Usage, see details at https://github.com/killbill/killbill/issues/839#issuecomment-358509758
* When creating multiple subscriptions (new quantity field on subscriptions introduced in 0.24) <- to be double checked

Reply:
Thanks, let me take a look 👍

Reply:
I will address this as part of overall corrections across all sections, merging the PR for now.